docs/tutorial/05-creating-modules.md

```diff
@@ -168,8 +168,27 @@ Since the `targetScope` has been changed, notice that the `$schema` property in
 
 Modules are a powerful way to separate your bicep files into logical units and abstract away complex resource declarations.
 
+## Using if condition with bicep Modules
+Modules will also support the if condition that was introduced in v0.2.212. The linked ARM template that will be created by the `bicep build` command will insert a condition at the deployment level for everything in the module.  To take advantage of this the module just needs to adhere to the existing syntax and usage of the if command when it is being called upon.  
```

Comment:
Your example is nice and self-explanatory, maybe you can sum the last sentence as:

To take advantage of this the module declaration needs to follow the if condition syntax: